action/index.js

```diff
@@ -86,7 +86,10 @@ async function main() {
   });
   core.debug(`Discovered ${assets.size} assets`);
 
-  const api = new API(`GitHubOIDC ${token}`, ORIGIN);
+  const api = new API(`GitHubOIDC ${token}`, ORIGIN, {
+    alwaysPrintDenoRay: true,
```

Comment:
I also wonder this needs to be an option if it's always set to `true`

Reply:
The intention here is that we don't turn on this option in deployctl CLI command.